osu.Game/Skinning/LegacySkin.cs

```diff
@@ -43,6 +43,7 @@ public class LegacySkin : Skin
         public LegacySkin(SkinInfo skin, IResourceStore<byte[]> storage, AudioManager audioManager)
             : this(skin, new LegacySkinResourceStore<SkinFileInfo>(skin, storage), audioManager, "skin.ini")
         {
+            if (!Configuration.CustomColours.ContainsKey("SliderBall")) Configuration.CustomColours["SliderBall"] = new Color4(2, 170, 255, 255);
```

Comment:
Maybe we ought to make a `LegacySkinConfiguration` instead? Because this is probably going to come up more in the future.

Reply:
This isn't so simple to do as these defaults should only be applied via this particular constructor.

Unless you have any ideas, I can't see a low-overhead path forward to making this work with the current structure, so maybe best leaving for a future PR.